dali/base/dasds.cpp

```diff
@@ -2780,6 +2780,17 @@ class CServerRemoteTree : public CRemoteTreeBase
         return child->hasProp(EXT_ATTR);
     }
 
+    virtual CompressionMethod getCompressionType() const override
+    {
+        CompressionMethod method = CRemoteTreeBase::getCompressionType();
+        if (method) return method;
+        if (SDSManager->ignoreExternals) return COMPRESS_METHOD_NONE;
+        CServerRemoteTree *child = (CServerRemoteTree *)queryPropTree(nullptr);
```

Comment:
can't see why the cast is needed here (same is true of pre-existing isCompressed above though).
Looks like the isCompressed 'chiild' could just be a IPropertyTree,
and this impl. could do away with the queryPropTree(nullptr) altogether, i.e. be simplified to:
```
if (hasProp(EXT_ATTR))
    return COMPRESS_METHOD_NONE;
```